src/sparseml/exporters/transforms/convert_quantizable_matmul.py

```diff
@@ -158,4 +157,5 @@ def transform(self, model: ModelProto) -> ModelProto:
                 f"Converted {count_converted_nodes} quantizable MatMul ops "
                 "to QLinearMatMul"
             )
+        ONNXGraph(model).sort_nodes_topologically()
```

Comment:
Why is this necessary here?

Reply:
Just moving it outside the loop so we are only doing it once instead of every time we match. not a bug fix, but an easy optimization